frontend/app_flowy/lib/workspace/presentation/settings/settings_dialog.dart

```diff
@@ -65,4 +57,19 @@ class SettingsDialog extends StatelessWidget {
       ),
     );
   }
+
+  Widget getSettingsView(int index, UserProfilePB user) {
```

Comment:
Could the bloc store the current view as an enum rather than an int?

Reply:
Sure. I have replaced the int type with the enum type.